demo/server/server_actions/Route.re

```diff
@@ -0,0 +1,81 @@
+// QUESTION: How should we create this manifest automatically?
```

Comment:
This is something we need to know before merging this PR, do you have a cool answer?

Reply:
Options:

- Have a "collector" module to register and get the actions by ID. (this one looks more error-proof and easy to apply)
- Have a ppx that generates those manifest functions.